config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :organisations
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "organisations#index"
```

Comment:
The changes to routes:
- line 2 sets up all the paths for devise to work (log in and out paths together with registration should we want to use it.
- line 3 sets up the CRUD paths for organisations.
- line 7 set the default "\" path to point at the organisation list.

Use `rails routes` to see what they look like. The `-g` (grep) option allows the list to be modified to only show routes with certain content. So `rails routes -g organisation` outputs:
```
           Prefix Verb   URI Pattern                       Controller#Action
    organisations GET    /organisations(.:format)          organisations#index
                  POST   /organisations(.:format)          organisations#create
 new_organisation GET    /organisations/new(.:format)      organisations#new
edit_organisation GET    /organisations/:id/edit(.:format) organisations#edit
     organisation GET    /organisations/:id(.:format)      organisations#show
                  PATCH  /organisations/:id(.:format)      organisations#update
                  PUT    /organisations/:id(.:format)      organisations#update
                  DELETE /organisations/:id(.:format)      organisations#destroy
             root GET    /                                 organisations#index
```